test/NuGet.Frameworks.Test/CompatibilityTests.cs

```diff
@@ -237,13 +249,37 @@ public void Compatibility_DotNetNeg(string framework)
         [InlineData("dnxcore")]
         [InlineData("netcore50")]
         [InlineData("netcore60")]
-        public void Compatibility_DotNetCompat(string framework)
+        [InlineData("uap")]
+        [InlineData("uap11.0")]
+        [InlineData("wpa")]
+        [InlineData("wpa81")]
+        [InlineData("netcore")]
+        [InlineData("win")]
+        [InlineData("win8")]
+        [InlineData("win81")]
+        [InlineData("monotouch")]
+        [InlineData("monotouch10")]
+        [InlineData("monoandroid")]
+        [InlineData("monoandroid40")]
+        [InlineData("monomac")]
+        [InlineData("xamarinios")]
+        [InlineData("xamarinmac")]
+        [InlineData("xamarinpsthree")]
+        [InlineData("xamarinpsfour")]
+        [InlineData("xamarinpsvita")]
+        [InlineData("xamarinxboxthreesixty")]
+        [InlineData("xamarinxboxone")]
+        [InlineData("sl6")]
```

Comment:
I think we want to block `sl` from `dotnet` entirely, rather than just up to `sl5`